### ISSUE TITLE
Provide some support for Helm v2

### ISSUE THREAD

**Issue by marckhouzam** [helm/acceptance-testing#20]
I'd like to allow for some tests to be run with Helm v2.  To achieve this it should no longer be `acceptance.sh` that aborts execution when detecting Helm v2 but it should instead be pushed down to each robot test suite to choose if it supports v2 or not.

**Comment by marckhouzam** [comment]
@jdolitsky If a test suite does not support v2, do you think its tests should be skipped with a PASSED status or FAILED status (there is no SKIPPED in Robot)?  I'm leaning towards a PASS so that the final result of the entire acceptance run does not always look like a failure for v2.

Skipped with PASS would look like this:
```
==============================================================================
Kubernetes Versions :: Verify Helm functionality on multiple Kubernetes ver...
==============================================================================
Helm works with Kubernetes 1.14.3                                     | PASS |
Helm v2 not supported. Skipping test.
------------------------------------------------------------------------------
Helm works with Kubernetes 1.15.0                                     | PASS |
Helm v2 not supported. Skipping test.
------------------------------------------------------------------------------
Kubernetes Versions :: Verify Helm functionality on multiple Kuber... | PASS |
2 critical tests, 2 passed, 0 failed
2 tests total, 2 passed, 0 failed
==============================================================================
```

Skipped with FAIL would look like this:
```
==============================================================================
Kubernetes Versions :: Verify Helm functionality on multiple Kubernetes ver...
==============================================================================
Helm works with Kubernetes 1.14.3                                     | FAIL |
Helm v2 not supported. Skipping test.
------------------------------------------------------------------------------
Helm works with Kubernetes 1.15.0                                     | FAIL |
Helm v2 not supported. Skipping test.
------------------------------------------------------------------------------
Kubernetes Versions :: Verify Helm functionality on multiple Kuber... | FAIL |
2 critical tests, 0 passed, 2 failed
2 tests total, 0 passed, 2 failed
==============================================================================
```

**Comment by jdolitsky** [comment]
@marckhouzam I think PASS is fine so long as we do have that output "skipping test"

**Comment by jdolitsky** [comment]
Fixed by #21
